### ISSUE TITLE
[Analytics] GA4 Events for Form Submission

### ISSUE THREAD

**Issue by micahchiang** [department-of-veterans-affairs/va-forms-system-core#554]
## Description
Google Analytics should track events around form submission. 

## Considerations
- Should be completed after #553
- All GA4 submission events are listed in the [GA4 analytics migration document in confluence](https://vfs.atlassian.net/wiki/spaces/FLT/pages/2248016008/Google+Analytics+4+Migration)
- vets-website [analytics helpers](https://github.com/department-of-veterans-affairs/vets-website/blob/a031382b8514e7a4e77761c12c24a2ffdd2d0ed2/src/platform/monitoring/record-event.js#L3)
- VAFSC may need some sort of analytics configuration, whoever is working on this should verify. (i.e. does VAFSC need to import a script tag for GA, or is something like `window.dataLayer && window.dataLayer.push` sufficient)
- Should be be able to verify locally that events are firing with `window.dataLayer`. 

## Tasks 
- [ ] Add events to onSubmit functionality completed in #553

## Acceptance Criteria
- [ ] Tasks have been complete
- [ ] PR has been merged into main
